index.js

```diff
@@ -82,6 +128,30 @@ module.exports = {
     return tree;
   },
 
+  postBuild({ directory }) {
+    if (!process.env.EMBER_CLI_ELECTRON) {
+      return tree;
+    }
+
+    //
+    // electron-forge needs our package.json to have a main entry pointing to
+    // electron.js. I can't figure out how to get broccoli to edit JSON for us,
+    // so we just do it manually once the build is complete.
+    //
```

Comment:
This is the `package.json` at the root of the project. I removed the code from the blueprint that sets `main` to `ember-electron/electron.js`, so new apps would not have it set (although yours probably does because you ran the blueprint back when it was setting `main`).

The reason I did this is because I don't think setting `main` in the ember project's `package.json` really makes sense. We never run `ember-electron/electron.js` out of the Ember project, we "build" a new node package by copying the `package.json`, `ember-electron/electron.js`, the `public` files and the Ember build output to a new location and then run from there. So saying that the Ember project's main routing is `ember-electron/electron.js` doesn't make sense because we don't (and can't) actually run that file.

Moreover, I could imagine situations where an Ember application also does double-duty as a simple node package, and since a package only gets one `main` script, it seems best not to set it if we really don't need to.